.github/workflows/ci.yml

```diff
@@ -67,6 +67,9 @@ jobs:
           components: clippy,rustfmt
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
+        with:
+          # The last version of Foundry that didn't experience this issue: https://github.com/alloy-rs/alloy/issues/1371
```

Comment:
We should open an issue to remind of addressing this later.

Reply:
https://github.com/FuelLabs/fuel-block-committer/issues/127